tests/integration_tests/datasets/commands_tests.py

```diff
@@ -250,6 +250,7 @@ def test_import_v0_dataset_cli_export(self):
         }
         command = v0.ImportDatasetsCommand(contents)
         command.run()
+        db.session.commit()
```

Comment:
It's a flaky test here. The `after_insert` in the `SqlaTable`  doesn't run `commit` finally, so the Postgresql probably does not preserve shadow tables.